src/setups/extrude_geometry.jl

```diff
@@ -31,45 +31,54 @@ Extrude either a line, a plane or a shape along a specific direction.
                         When `tlsph=true`, particles will be placed on the boundary of the shape.
 
 # Examples
-```julia
-# 2D
+```jldoctest; output = false, filter = [r"┌ Info: The desired size .*\n", r"└ New particle spacing is set to .*\n"]
+# Extrude a line in 2D to a plane in 2D
 p1 = [0.0, 0.0]
 p2 = [1.0, 1.0]
 
 direction = [-1.0, 1.0]
 
-shape = ExtrudeGeometry((p1, p2); direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
+shape = extrude_geometry((p1, p2); direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
 
-# 3D Plane
+# Extrude a parallelogram in 3D to a parallelepiped in 3D
 p1 = [0.0, 0.0, 0.0]
 p2 = [0.5, 1.0, 0.0]
 p3 = [1.0, 0.2, 0.0]
 
 direction = [0.0, 0.0, 1.0]
 
-shape = ExtrudeGeometry((p1, p2, p3); direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
+shape = extrude_geometry((p1, p2, p3); direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
 
-# Extrude a 2D shape to a 3D shape
+# Extrude a 2D shape (here: a disc) to a 3D shape (here: a cylinder)
 shape = SphereShape(0.1, 0.5, (0.2, 0.4), 1000.0, n_layers=3,
                     sphere_type=RoundSphere(end_angle=pi))
 
 direction = [0.0, 0.0, 1.0]
 
-shape = ExtrudeGeometry(shape; direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
+shape = extrude_geometry(shape; direction, particle_spacing=0.1, n_extrude=4, density=1000.0)
+
+# output
```

Comment:
Why not remove the filter above (idk why it's not working) and explicitly test for this output?
```suggestion
# output
┌ Info: The desired size is not a multiple of the particle spacing 0.1.
└ New particle spacing is set to 0.09387239731236392.
┌ Info: The desired size is not a multiple of the particle spacing 0.1.
└ New particle spacing is set to 0.09198039027185569.
```